src/common/components/navigation/Menu.tsx

```diff
@@ -82,6 +110,22 @@ export interface MenuProps {
 
 export const Menu: React.FunctionComponent<MenuProps> = (props) => {
   const classes = useStyles();
+  const { url } = useRouteMatch();
+  const { pathname } = useLocation();
+
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  const handleSetSelectedIndex = () => {
+    menuItems.map((item, i) => {
+      if (pathname === `${url}${item.path}`) {
+        setSelectedIndex(i);
+      }
+    });
```

Comment:
It may make more sense to use `findIndex()` instead of `map()`, and use that as new value for `selectedIndex`. Also, currently a warning is thrown.